### PR TITLE
ReleaseNotes: Updated changelog and release notes for 8.2.0-beta2

### PR DESCRIPTION
**Pull request by grafanabot**
This exciting! So much has changed!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,58 @@
+
+<!-- 8.2.0-beta2 START -->
+
+# 8.2.0-beta2 (2021-09-30)
+
+### Features and enhancements
+
+- **AccessControl:** Document new permissions restricting data source access. [#39091](https://github.com/grafana/grafana/pull/39091), [@gamab](https://github.com/gamab)
+- Add fiscal years and search to time picker. [#39073](https://github.com/grafana/grafana/pull/39073), [@oscarkilhed](https://github.com/oscarkilhed)
```
Comment (osg-grafana) [inline CHANGELOG.md:9]: What is the prefix for this feature or enhancement?

Comment (ivanahuckova) [reply CHANGELOG.md:8]:
```suggestion
- **TimePicker:** Add fiscal years and search to time picker. [#39073](https://github.com/grafana/grafana/pull/39073), [@oscarkilhed](https://github.com/oscarkilhed)
```

Comment (gabor) [reply CHANGELOG.md]: @osg-grafana based on the PR it is a change to timepicker.

**Review by gabor — APPROVED** [review]
considering the time-pressure i am approving this, it looks ok.

**Pull request merged**